### DISCUSSION TITLE
Need a Q&A robot for next.js?

### DISCUSSION THREAD

**Discussion by xiaofan-luan** [vercel/next.js#48683] [Show and tell]
We have developed OSSChat, a free chatbot that helps you learn more about open source software. It is powered by ChatGPT and vector databases such as Milvus.

We are pleased to announce that we now support Next.Js. Feel free to ask any questions about Next.Js, and OSSChat will help you find relevant documents to assist you. Enjoy!

You can access OSSChat at https://osschat.io/.

**Comment by xiaofan-luan** [comment]
Hopefully, this could help the maintainer of the community with very basic topics. I understand how difficult it is to maintain a prosperous community.

**Reply by 3CE8D2BAC65BDD6AA9** [reply]
I am very impressed by the OSSChat.  Where can I find the source code of the site?

**Reply by xiaofan-luan** [reply]
check here zilliztech/akcio
